wom/src/test/scala/wom/types/WomTypeSpec.scala

```diff
@@ -0,0 +1,167 @@
+package wom.types
+
+import org.scalatest.prop.TableDrivenPropertyChecks._
+import org.scalatest.prop.Tables.Table
+import org.scalatest.{FlatSpec, Matchers}
+import spray.json.JsString
+import wom.values._
+
+import scala.runtime.ScalaRunTime
+
+class WomTypeSpec extends FlatSpec with Matchers {
+  "WdlType class" should "stringify WdlBoolean to 'Boolean'" in {
+    WomBooleanType.toWdlString shouldEqual "Boolean"
```

Comment:
Is renaming this to `toWomString` part of your other commit?

Reply:
Also the `wdlType` method on `WomValue`s?

Reply:
I actually renamed `toWdlString` to `toDisplayString`. I always hated that `toWdlString` name since it didn't return a `WdlString` at all, but rather a readable name that we would use in simpletons and other places.

Reply:
fixed the `wdlType` => `womType` too

Reply:
"since it didn't return a WdlString at all"

You probably just heard my deep sigh all the way from Orlando.